crates/orchestrator/api.toml

```diff
@@ -3,15 +3,6 @@ NAME = "orchestrator"
 DESCRIPTION = "Orchestrator for HotShot"
 FORMAT_VERSION = "0.1.0"
 
-# POST node's identity
-[route.postidentity]
-PATH = ["identity/:identity"]
-METHOD = "POST"
-":identity" = "Literal"
-DOC = """
-POST a node's identity (IP address) to the orchestrator.  Returns the node's node_index
-"""
-
```

Comment:
Why this endpoint is removed? The `node_index` it returns will help with getting validator's config.